### PR TITLE
Home and Add Apk page added

### PR DESCRIPTION
**Pull request by BINAH25**
Apk Upload page added
image
Home page to download apk added
image

### CODE REVIEW

File: web/src/pages/HomePage/style.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@import "../../theme-variables.scss";
+
+.home-page {
+    height: 100vh;
+    background: linear-gradient(-45deg, #000 0%, #052892 100%);
```
Comment (dodziraynard) [inline web/src/pages/HomePage/style.scss:5]: Let's change the colour to Y-Check colours.
You can also do your own design if you prefer.

File: web/src/widgets/ApkWidget/index.jsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    
+    return (
+        <Fragment>
+            <form className="row bio-data p-3" onSubmit={handleFormSubmit}>
+               
+            {webConfigurations?.version
+                && <p><a className='badge bg-primary'>App Current Version: {webConfigurations.version}</a></p>
+            }
+                <div className="mb-3 col-md-12">
+                    <label htmlFor="formFile" className="form-label">Upload apk file</label>
```
Comment (dodziraynard) [inline web/src/widgets/ApkWidget/index.jsx:90]: Also show the download link on the setup page as shown here.
image

File: web/src/widgets/SetupWidget/index.jsx [hunk 3]
```diff
@@ -13,6 +13,7 @@ function SetupWidget() {
                     <NavLink className="setup-menu-item" to="facilities"><i className="bi bi-h-circle"></i> Facilities</NavLink>
                     <NavLink className="setup-menu-item" to="services"><i className="bi bi-gear"></i> Services</NavLink>
                     <NavLink className="setup-menu-item" to="roles"><i className="bi bi-lock"></i> Roles</NavLink>
+                    <NavLink className="setup-menu-item" to="apk"><i className="bi bi-app"></i> APK</NavLink>
```
Comment (dodziraynard) [inline web/src/widgets/SetupWidget/index.jsx:16]: Change APK to System

**Review by dodziraynard — CHANGES_REQUESTED** [review]
Well! Just fix the little changes in the comment.

**Comment by BINAH25** [comment]
okay sure

File: web/src/widgets/ApkWidget/index.jsx (hunk 2 above)
Comment (BINAH25) [reply web/src/widgets/ApkWidget/index.jsx:90]: done
image

File: web/src/widgets/SetupWidget/index.jsx (hunk 3 above)
Comment (BINAH25) [reply web/src/widgets/SetupWidget/index.jsx:16]: done

File: web/src/pages/HomePage/style.scss (hunk 1 above)
Comment (BINAH25) [reply web/src/pages/HomePage/style.scss]: image

**Pull request merged**